build-scripts/builder/windows/target/android.bat

```diff
@@ -0,0 +1,27 @@
+echo off
```

Comment:
Is this file used anywhere ?

Reply:
I decided to move the actual contents of that into a separate PR as this one was getting rather big.